### PR TITLE
adds velaux sql database docs

### PR DESCRIPTION
**Pull request by NeerajGartia21**
### Description of your changes

Adds documentation for mysql and postgresql databases of VelaUX.

Screenshot 2023-09-07 at 3 20 53 PM

I have:

- [X] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [X] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: docs/platform-engineers/system-operation/high-availability.md [hunk 1]
```diff
@@ -39,8 +39,9 @@ kubectl scale deploy -n vela-system kubevela-cluster-gateway --replicas=3
 
 VelaUX is an addon, it composed by 3 major parts: nginx for frontend, APIServer as backend, and its database.
 
-We strongly recommend to use high availability mongodb cluster as the database driver. The frontend and backend are both stateless services, just configure multiple replicas for them. As a result, the high availability installation command just like below:
+We strongly recommend to use high availability mongodb, mysql or postgresql cluster as the database driver. The frontend and backend are both stateless services, just configure multiple replicas for them. As a result, the high availability installation command just like below:
```
Comment (chivalryq) [inline docs/platform-engineers/system-operation/high-availability.md]:
```suggestion
We strongly recommend to use high availability mongodb, mysql(beta) or postgresql(beta) cluster as the database driver. The frontend and backend are both stateless services, just configure multiple replicas for them. As a result, the high availability installation command just like below:
```

File: docs/reference/addons/velaux.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 title: VelaUX
 ---
 
+import Tabs from '@theme/Tabs';
```
Comment (chivalryq) [inline docs/reference/addons/velaux.md:5]: If I remember it correctly, this feature is supported by mdx file. Have you preview the changes?

File: docs/reference/addons/velaux.md [hunk 3]
```diff
@@ -173,6 +179,104 @@ spec:
...
+  workflow:
+    steps:
+      - name: vela-system
+        properties:
+          policies:
+            - vela-system
+        type: deploy
+```
+
+After deployed, let's get the root password from the secret `vela-system/velaux-db-postgresql`.
```
Comment (chivalryq) [inline docs/reference/addons/velaux.md:277]: how to get the password? Give a shell command?

File: docs/reference/addons/velaux.md (hunk 2 above)
Comment (NeerajGartia21) [reply docs/reference/addons/velaux.md:5]: Yes, I have previewed the changes and it was working fine.

File: docs/reference/addons/velaux.md (hunk 3 above)
Comment (NeerajGartia21) [reply docs/reference/addons/velaux.md:277]: Yes I think

File: docs/reference/addons/velaux.md (hunk 2 above)
Comment (chivalryq) [reply docs/reference/addons/velaux.md:5]: Please attach it in the PR descriptions. That's always good for reviewers to understand the effect.

Comment (NeerajGartia21) [reply docs/reference/addons/velaux.md:5]: Oh sorry. I missed it. I will attach

Comment (NeerajGartia21) [reply docs/reference/addons/velaux.md:5]: Attached.

File: docs/reference/addons/velaux.md (hunk 3 above)
Comment (chivalryq) [reply docs/reference/addons/velaux.md:277]: @NeerajGartia21 How is this going?

Comment (NeerajGartia21) [reply docs/reference/addons/velaux.md:277]: I'm not able to fetch the secret in my local env. Can you please give it a try?

**Review by chivalryq — APPROVED** [review]

**Pull request merged**